### ISSUE TITLE
Invalid links in docs

### ISSUE THREAD

**Issue by caesarxuchao** [kubernetes/kubernetes#21829]
I've set up a Jenkins job to verify the links in our docs daily. The following is a list of invalid links it found. I'll try to fix them myself and reach out to folks if I can't figure out what the correct link is.

```
14:00:37 Summary of invalid links:
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/pkg/apis/extensions/v1beta1/types.go, Got 404 visiting http://releases.k8s.io/HEAD/docs/admin/daemon.md
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/pkg/apis/extensions/v1beta1/types.go, Got 404 visiting http://docs.k8s.io/api-conventions.md#metadata
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/admin/kubelet.md, Got 404 visiting http://releases.k8s.io/HEAD/docs/user-guide/compute-resources.html
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/design/event_compression.md, Got 404 visiting http://releases.k8s.io/HEAD/pkg/client/unversioned/record/event.go
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/devel/on-call-build-cop.md, Got 404 visiting https://github.com/kubernetes/contrib/tree/master/submit-queue
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/devel/on-call-build-cop.md, Got 404 visiting https://github.com/kubernetes/kubernetes/blob/doc-flaky-test/docs/devel/flaky-tests.md#filing-issues-for-flaky-tests
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/devel/on-call-user-support.md, Got 404 visiting https://github.com/k8s-support-oncall
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/kubernetes-release/release/v0.15.0/bin/linux/amd64/kubecfg
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/k8s/flanneld
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/docker-multinode/master.md, Got 404 visiting http://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-hadoop-on-mesos-using-installer
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-spark-on-mesos
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-chronos-on-mesos
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/ovirt.md, Got 404 visiting http://www.ovirt.org/How_to_install_the_guest_agent_in_Fedora
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/rackspace.md, Got 404 visiting https://github.com/kubernetes/kubernetes/archive/v0.9.tar.gz
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/scratch.md, Got 404 visiting http://releases.k8s.io/HEAD/cluster/addons/kube-ui/
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/vsphere.md, Got 404 visiting https://storage.googleapis.com/govmomi/vmdk/2016-01-08/kube.vmdk.gz{,.md5}
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/proposals/security-context-constraints.md, Got 404 visiting http://docs.k8s.io/design/service_accounts.md
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/proposals/workflow.md, Got 404 visiting https://github.com/kubernetes/kubernete2s/pull/11980
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/user-guide/configuring-containers.md, Got 404 visiting https://docs.docker.com/userguide/usingdocker/
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/user-guide/logging.md, Got 404 visiting http://releases.k8s.io/release-1.1/contrib/logging/fluentd-sidecar-gcp/README.md
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/user-guide/production-pods.md, Got 404 visiting http://releases.k8s.io/HEAD/contrib/git-sync/
```

@lavalamp @bgrant0607 for triage.

**Comment by bgrant0607** [comment]
cc @johndmulhausen

**Comment by caesarxuchao** [comment]
In docs/devel/on-call-user-support.md, the link in line `[@k8s-support-oncall](https://github.com/k8s-support-oncall) will reach the current person on call.` is invalid. I cannot track the author of that file. Do you know what the correct link is or who is responsible for that file? Thanks. @lavalamp @bgrant0607

**Comment by caesarxuchao** [comment]
I'll ignore these two links since the enclosing doc seems to be out dated:
```
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/kubernetes-release/release/v0.15.0/bin/linux/amd64/kubecfg
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/k8s/flanneld
```

**Comment by caesarxuchao** [comment]
Can we find some mesos guys to fix these links in the doc?
```
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/kubernetes-release/release/v0.15.0/bin/linux/amd64/kubecfg
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/coreos/bare_metal_offline.md, Got 404 visiting https://storage.googleapis.com/k8s/flanneld
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-hadoop-on-mesos-using-installer
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-spark-on-mesos
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/mesos.md, Got 404 visiting http://mesosphere.com/docs/tutorials/run-chronos-on-mesos
```

**Comment by caesarxuchao** [comment]
@simon3z I think you create the ovirt.md, could you help fix this link?
```
14:00:37 Failed: in file /jenkins-master-data/jobs/kubernetes-test-linkchecker/workspace/docs/getting-started-guides/ovirt.md, Got 404 visiting http://www.ovirt.org/How_to_install_the_guest_agent_in_Fedora
```

**Comment by simon3z** [comment]
@caesarxuchao the page was moved to:

http://www.ovirt.org/documentation/how-to/guest-agent/install-the-guest-agent-in-fedora/

**Comment by caesarxuchao** [comment]
Thanks @simon3z. Fixed in #21927.

**Comment by caesarxuchao** [comment]
Mostly fixed in #21927. Will reopen when we have more invalid links.
